src/sentry/search/snuba/backend.py

```diff
@@ -131,6 +131,26 @@ def apply(self, name, parameters):
         )
 
 
+def get_search_filter(search_filters, name, operator):
+    """
+    Finds the value of a search filter with the passed name and operator. If
+    multiple values are found, returns the most restrictive value
+    :param search_filters: collection of `SearchFilter` objects
+    :param name: Name of the field to find
+    :param operator: '<' or '>'
+    :return: The value of the field if found, else None
+    """
+    comparator = max if operator.startswith('>') else min
```

Comment:
If someone passes `=` or `==` is `min` a good function to use?

Reply:
It's intended to only accept `'<'` or `'>'`, I could guard against anything else with an assert